### ISSUE TITLE
DeprecationWarning for LooseVersion usage

### ISSUE THREAD

**Issue by macintacos** [Teemu/pytest-sugar#227]
This is mostly just a "heads up", using this with Python 3.10, I'm getting a Deprecation warning that looks like the following:

```
/Users/juliant/Library/Caches/pypoetry/virtualenvs/ops-template-wDb_TPtP-py3.10/lib/python3.10/site-packages/pytest_sugar.py:168: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
  from distutils.version import LooseVersion
```

It's pretty noisy in my test output, hope it can be fixed.

**Comment by decaz** [comment]
Solution can be found here: https://github.com/marshmallow-code/marshmallow/commit/a63c28edde1e3f1f522aab8a7de231230786ebc1.

**Comment by kbroch-rivosinc** [comment]
I'm looking for a workaround to disable this warning and wondering if anyone has done it?

[This](https://docs.pytest.org/en/6.2.x/warnings.html#deprecationwarning-and-pendingdeprecationwarning) appears to only be related to warnings in the user code.  I tried it anyways and didn't change output.

Other failed attempts:

* `pytest --verbose --disable-pytest-warnings`
* `pytest --verbose --disable-warnings`
* `pytest --verbose -W ignore:pytest_sugar:DeprecationWarning` (probably same results as putting in `pytest.ini`)

Am I missing something or is this just not possible?

**Comment by Cielquan** [comment]
Fixed by #230
